components/google_sheets/google_drive.app.js

```diff
@@ -0,0 +1,235 @@
+const axios = require("axios");
```

Comment:
this looks like a copy paste of https://github.com/PipedreamHQ/pipedream/blob/master/components/google_drive/google_drive.app.js
and it's out of date now.  Remove this file and just require the one in the google_drive directory.

Reply:
Much of it is copied from the one in the google_drive directory, but the last 3 methods are not. Maybe I could require the one in the google_drive directory in this app file, remove the copied methods, and reference the original methods (from the google_drive directory app file) in this app file?